setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='sensu_plugin',
-    version='0.4.5',
+    version='0.4.6',
```

Comment:
I generally recommend bumping versions outside of PRs to avoid needing to rebase more often. If you are then you should update the CHANGELOG to reflect.

Reply:
good point, I'll fix that now.